handlers/user.js

```diff
@@ -0,0 +1,598 @@
+ *          schema:
+ *           type: object   
+ *           properties: 
+ *                  username:
+ *                     type: string
+ *                  first_name:
+ *                     type: string
+ *                  last_name:
+ *                     type: string
+ *                  age:
+ *                     type: integer
+ *                  email:
+ *                     type: string
+ *                  password:
+ *                     type: string
+ *     responses:
+ *       200:
+ *        description: Sign Up.
+ *        content:
+ *          application/json:
+ *             schema: 
+ *                type: object
+ *                properties:
+ *                  id:
+ *                     type: integer
+ *                  username:
+ *                     type: string
+ *                  first_name:
+ *                     type: string
+ *                  last_name:
+ *                     type: string
+ *                  age:
+ *                     type: integer
+ *                  email:
+ *                     type: string
+ *                  password:
+ *                     type: string
+ *       404:
+ *          description: Impossible to complete sign up.
+ *          content:
+ *            application/json:
+ *              schema: 
+ *                type: string
+ */
+user.post('/signup', validate([
+    body('username').isLength({ max: 30 }),
+    body('first_name').isLength({ max: 15 }),
+    body('last_name').isLength({ max: 15 }),
+    body('age').isNumeric().isLength({ max: 3 }),
+    body('email').isEmail().isLength({ max: 60 }),
+    body('password').isLength({ max: 30 }),
+]),
+    async (request, response) => {
+        try {
+            const user = await userController.signUp(request.body);
+            response.json(user);
+        } catch (error) {
+            request.log.error(error);
+            response.end(JSON.stringify(error)).status(500);
+        }
+    });
+
+
+/**
+* @openapi
+* /users/profile:
+*   post:
+*     description: Authentication.
+*     tags:
+*      - User
+*     produces:
+*       - application/json
+*     security:
+*      - bearerAuth: []      
+*     responses:
+*       200:
+*        description: Login.
+*        content:
+*          application/json:
+*             schema: 
+*                type: object
+*                properties:
+*                  id:
+*                     type: integer
+*                  username:
+*                     type: string
+*                  first_name:
+*                     type: string
+*                  last_name:
+*                     type: string
+*                  age:
+*                     type: integer
+*                  email:
+*                     type: string
+*       401:
+*          description: Unauthorized.
+*          content:
+*            application/json:
+*              schema: 
+*                type: string
+*/
+user.post('/profile', authentication, (request, response) => {
```

Comment:
I think this was your initial testing method in the previous implementation. You don't need it anymore.